html/statbrowser.html

```diff
@@ -0,0 +1,254 @@
+		draw_spells(tab)
+	} else {
+		document.body.textContent = "...";
+	}
+	if(tab == "Status" || tab == "MC" || spell_tabs_thingy) {
+		window.location.href = "byond://winset?statbrowser.is-visible=true";
+	} else {
+		window.location.href = "byond://winset?statbrowser.is-visible=false";
+	}
+}
+
+function draw_status() {
+	document.body.innerHTML = "";
+	for(let i = 0; i < status_tab_parts.length; i++) {
+		if(status_tab_parts[i].trim() == "") {
+			document.body.appendChild(document.createElement("br"));
+		} else {
+			let div = document.createElement("div");
+			div.innerHTML = status_tab_parts[i];
+			document.body.appendChild(div);
+		}
+	}
+}
+
+function draw_mc() {
+	document.body.innerHTML = "";
+	let table = document.createElement("table");
+	for(let i = 0; i < mc_tab_parts.length; i++) {
+		let part = mc_tab_parts[i];
+		let tr = document.createElement("tr");
+		let td1 = document.createElement("td");
+		td1.textContent = part[0];
+		let td2 = document.createElement("td");
+		if(part[2]) {
+			let a = document.createElement("a");
+			a.href = "?_src_=vars;admin_token=" + href_token + ";Vars=" + part[2];
+			a.textContent = part[1];
+			td2.appendChild(a);
+		} else {
+			td2.textContent = part[1];
+		}
+		tr.appendChild(td1);
+		tr.appendChild(td2);
+		table.appendChild(tr);
+	}
+	document.body.appendChild(table);
+}
+
+function draw_spells(cat) {
+	document.body.innerHTML = "";
+	let table = document.createElement("table");
+	for(let i = 0; i < spells.length; i++) {
+		let part = spells[i];
+		//if(part[0] != cat) continue;
+		let tr = document.createElement("tr");
+		let td1 = document.createElement("td");
+		td1.textContent = part[1];
+		let td2 = document.createElement("td");
+		if(part[3]) {
+			let a = document.createElement("a");
+			a.href = "?src=" + part[3] + ";click=1";
+			a.textContent = part[2];
+			td2.appendChild(a);
+		} else {
+			td2.textContent = part[2];
+		}
+		tr.appendChild(td1);
+		tr.appendChild(td2);
+		table.appendChild(tr);
+	}
+	document.body.appendChild(table);
+}
+
+function set_light_theme() {
+	document.body.style.backgroundColor = "white";
+	document.body.style.color = "black";
+	document.body.className = "";
+	set_style_sheet("browserOutput_white")
+}
+
+function set_dark_theme() {
+	document.body.style.backgroundColor = "#171717";
+	document.body.style.color = "#a4bad6";
+	document.body.className = "dark";
+	set_style_sheet("browserOutput")
+}
+
+function set_style_sheet(sheet) {
+	if(document.getElementById("goonStyle")) {
+		let currentSheet = document.getElementById("goonStyle");
+		currentSheet.parentElement.removeChild(currentSheet);
+	}
+	let head  = document.getElementsByTagName('head')[0];
+	let sheetElement = document.createElement("link");
+	sheetElement.id = "goonStyle";
+	sheetElement.rel = "stylesheet";
+	sheetElement.type = "text/css";
+	sheetElement.href = sheet + ".css";
+	sheetElement.media = 'all';
+	head.appendChild(sheetElement);
+}
+
+window.addEventListener("load", function() {
```

Comment:
This will throw an error for IE8 users, which is fine imo.
I would add that we dropped support for IE8 into the changelog though, it's something players should be aware of.